geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/hash/AbstractHashesIntegrationTest.java

```diff
@@ -48,6 +50,7 @@
   private Jedis jedis;
   private Jedis jedis2;
   private static int ITERATION_COUNT = 4000;
+  private static int NUM_ITERATIONS = 1000;
```

Comment:
Having a constant named "ITERATION_COUNT" right next to a constant named "NUM_ITERATIONS" feels weird. At least one of them should be renamed to be a bit more specific.

Reply:
👍 good call. NUM_ITERATIONS was only used once so I just got rid of it